cli/generators/crud/crudGenerator.js

```diff
@@ -0,0 +1,23 @@
+var Generator = require('yeoman-generator');
+var ejs = require('ejs');
+
+module.exports = class extends Generator {
+  constructor(args, opts) {
+    super(args, opts);
+  }
+
+  appendTpl(templatePath, destinationPath, data) {
+    var fs = this.fs;
+    ejs.renderFile(templatePath, data, (err, renderedTemplate) => {
+      if(err) throw err;
+
+      fs.copy(
+        destinationPath,
```

Comment:
Hey @pbarrios, would you mind sending me some info about this recommendation? I tried searching around to learn in which cases this was required, but I couldn't find much.